<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Engatar Veículo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botaoSalvar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Engatar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrFrota">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Informe a Frota</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrConjunto">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Informe o Conjunto</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtEngate">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2024</year>
      <month>4</month>
      <day>10</day>
     </datetime>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoSalvarDesengate">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Desengatar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nº Frota</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nº Conjunto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="Fundo_2">
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>-110</y>
      <width>1121</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../fundDentroCaminhaommm.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrPlacaFrota">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 255, 183);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Placa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrPlacaA">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 255, 183);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Placa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrPlacaB">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 255, 183);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Placa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>ENGATE DE VEÍCULOS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tpVeic">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 255, 183);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Tração</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Tipo Veículo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VoltarMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
    border: none;
    color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/fast_backward_double_chevron_back_previous_left_icon_258776.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrPlacaC">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 255, 183);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Placa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrPlacaD">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 255, 183);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Placa</string>
    </property>
   </widget>
   <widget class="QLabel" name="fundo">
    <property name="geometry">
     <rect>
      <x>-220</x>
      <y>-150</y>
      <width>1131</width>
      <height>651</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/fundDentroCaminhaommm.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Placa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/documento.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoConsultaEngate">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
    border: none;
    color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>fundo</zorder>
   <zorder>Fundo_2</zorder>
   <zorder>nrFrota</zorder>
   <zorder>nrConjunto</zorder>
   <zorder>dtEngate</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>nrPlacaFrota</zorder>
   <zorder>nrPlacaA</zorder>
   <zorder>nrPlacaB</zorder>
   <zorder>label_11</zorder>
   <zorder>tpVeic</zorder>
   <zorder>label_7</zorder>
   <zorder>label</zorder>
   <zorder>nrPlacaC</zorder>
   <zorder>nrPlacaD</zorder>
   <zorder>botaoSalvarDesengate</zorder>
   <zorder>botaoSalvar</zorder>
   <zorder>label_6</zorder>
   <zorder>VoltarMenu</zorder>
   <zorder>label_2</zorder>
   <zorder>botaoConsultaEngate</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
